### PR TITLE
docs: restructure README

### PR DESCRIPTION
**Pull request by saccarosium**
Hi,
this PR tries to restructure the PR to be more legible and usable by leveraging some of GF Markdown features such as admonitions and tables.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,35 +20,52 @@ Distribution Alternatives:
...
+
+| OS | PATH |
+| :- | :--- |
+| Linux | `~/.config/nvim` |
+| MacOS | `~/.config/nvim` |
+| Windows | `%userprofile%\AppData\Local\nvim\` |
+
+#### Archive Installation
+
+* On the home/landing page for the project find the green "`<> CODE`" button click it and select `Local > Download ZIP`.
```
Comment (feoh) [inline README.md:41]: Hi. I can appreciate that you're trying to simplify here but I disagree with this change and feel it represents a negative departure from the current instructions.

If people use Git to clone the repo locally, they can then use `git pull` to update the repo from upstream as changes happen.

With your method, you've stripped that capability away which I see as a dis-service to new users.

File: README.md [hunk 2]
```diff
@@ -20,35 +20,52 @@ Distribution Alternatives:
...
+* Extract the archive to the appropriate configuration path.
+* Ensure your extraction method did not extract with a parent folder. For example in `~/.config/nvim` you should have `init.lua` not another folder called `kickstart.nvim`.
+
+#### Git Clone Installation
+
+From a terminal `cd`/`dir` to the configuration path and then run:
```
Comment (feoh) [inline README.md:47]: What does `cd`/`dir` mean? Do you mean 'cd or change directory? Or similar? Either way this isn't clear.

File: README.md [hunk 3]
```diff
@@ -20,35 +20,52 @@ Distribution Alternatives:
...
+  * [ripgrep](https://github.com/BurntSushi/ripgrep#installation) is required for multiple [telescope](https://github.com/nvim-telescope/telescope.nvim#suggested-dependencies) pickers.
+* See [Windows Installation](#Windows-Installation) if you have trouble with `telescope-fzf-native`
+
+Neovim's configurations are located under the following paths, depending on your OS:
+
+| OS | PATH |
+| :- | :--- |
+| Linux | `~/.config/nvim` |
+| MacOS | `~/.config/nvim` |
+| Windows | `%userprofile%\AppData\Local\nvim\` |
```
Comment (feoh) [inline README.md:37]: Nice change! Love this formatting.

File: README.md (hunk 1 above)
Comment (saccarosium) [reply README.md:41]: FWIW I've just used what was already on the README. We are actually on the same page here, I will much prefer that git was the preferred, and frankly only, option for installing kickstart

File: README.md (hunk 2 above)
Comment (feoh) [reply README.md:47]: I agree. Can you please go ahead and make that fix? (Or whoever authored this PR? :)

File: README.md (hunk 1 above)
Comment (feoh) [reply README.md:41]: OK thank you that makes sense. Wanna fix it and replace the unzip instructions with git clone instructions? :)

**Comment by feoh** [comment]
I'm going to merge this as is and we can iterate more with another PR if folks are willing :)

Thanks for the contribution!

**Pull request merged**